src/chrome/content/rules/GMX.xml

```diff
@@ -134,12 +123,10 @@
 		<test url="http://www.gmx.ch/" />
 		<test url="http://www.gmx.co.uk/" />
 		<test url="http://www.gmx.fr/" />
+		<test url="http://www.gmx.de/" />
```

Comment:
These test URLs are unnecessary: They are implicitly generated by the `<target host ...>` tags.